packages/apollo-links/src/authHttpLink.ts

```diff
@@ -68,6 +70,7 @@ function authHttpLink(options: AuthOptions): AuthHttpLink {
     authUrl,
     projectId: deploymentId,
     projectType,
+    indexers: indexers ?? [],
```

Comment:
I will prefer to using `undefined` for the way to disable this manual indexer picking.
```suggestion
    indexers,
```